plugin.d.ts

```diff
@@ -0,0 +1,20 @@
+// Definitions by: ricardo-devis-agullo <https://github.com/ricardo-devis-agullo>
```

Comment:
Do we really need this comment?

Reply:
Nope, I just copied from other fastify projects that had similar annotations. Can remove it.

Reply:
Please do, thanks!